uint8 camID
geometry_msgs/Pose2D pose